inputs.tf

```diff
@@ -29,3 +29,9 @@ variable "required_tags" {
   type        = map(any)
   default     = {}
 }
+
+variable "tags" {
```

Comment:
Strangely, this doesn't seem to be named "input_tags", as the README would indicate?  Am I seeing that correctly?